common/deposit_contract/build.rs

```diff
@@ -35,7 +35,7 @@ fn testnet_url() -> String {
 fn main() {
     match get_all_contracts() {
         Ok(()) => (),
-        Err(e) => panic!(e),
+        Err(e) => std::panic::panic_any(e),
```

Comment:
I think you must be using a Nightly compiler, this API isn't stable: https://doc.rust-lang.org/std/panic/fn.panic_any.html

Reply:
ah yeah. I forgot I had to switch to nightly to build some solidity tools :|